hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/impl/DefaultSqlClientPoolConfiguration.java

```diff
@@ -305,6 +305,23 @@ private String oraclePath(URI uri) {
 		return string.substring( start, end );
 	}
 
+	// if the host contains invalid characters
+	// {@link java.net.URI} parseAuthority(int start, int n) will end up setting host to NULL
```

Comment:
There's no point to use `{@link ...`} if it's not a javadoc.
Maybe in this case a javadoc would be better even if it's a private method.